### PR TITLE
Working get all endpoint with example data

### PR DESCRIPTION
**Pull request by PawelKepka**
url/api/games GET endpoint reutrns back example simple data without DB connection (in progress)

### CODE REVIEW

**Comment by DorianKossowski** [comment]
@Szy11mon please approve

**Review by Szy11mon — DISMISSED** [review]

**Comment by PawelKepka** [comment]
Connection to DB added

File: product-catalog-service/src/main/java/agh/fis/product_catalog/model/Product.java [hunk 1]
```diff
@@ -7,31 +7,31 @@
...
+import java.sql.Date; 
 
 @Entity
 public class Product extends ProductBaseEntity{
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private int id;
 
-    @Column(nullable = false)
+    @Column(nullable = false, columnDefinition="TEXT")
```
Comment (DorianKossowski) [inline product-catalog-service/src/main/java/agh/fis/product_catalog/model/Product.java:18]: I think `columnDefinition="TEXT"` is unnecessary, isn't it?

Comment (PawelKepka) [reply product-catalog-service/src/main/java/agh/fis/product_catalog/model/Product.java:18]: It is necessary in order to have string longer than 256, however it still doesnt work

**Comment by DorianKossowski** [comment]
@Szy11mon please approve

**Review by Szy11mon — DISMISSED** [review]

**Review by Szy11mon — APPROVED** [review]

**Review by DorianKossowski — APPROVED** [review]

**Pull request merged**